### ISSUE TITLE
Flutter 3.27.0: A value of type 'TabBarThemeData' can't be returned from a function with return type 'TabBarTheme'

### ISSUE THREAD

**Issue by benjamindean** [nank1ro/awesome_flutter_extensions#7]
Some theming classes have been removed in Flutter 3.27.0, therefore the following error is thrown during build:

```
A value of type 'TabBarThemeData' can't be returned from a function with return type 'TabBarTheme'.
```

**Comment by nank1ro** [comment]
Thanks for submitting the issue, will fix it very soon

**Comment by nank1ro** [comment]
fixed in the v1.3.0
